site2/website/release-notes.md

```diff
@@ -1,6 +1,207 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.8.2
+
+#### 2022-03-10
+
+### Important Notices
+
+- Fixed detecting number of NICs in EC2 [#14252](https://github.com/apache/pulsar/pull/14252). In the event that Pulsar cannot determine the NIC speed from the host, please set `loadBalancerOverrideBrokerNicSpeedGbps`.
```

Comment:
suggest using [present tense imperative](https://web.mit.edu/course/21/21.guide/m-impera.htm#:~:text=The%20imperative%20mood%20is%20used,base%20form%20of%20the%20verb.)